### PR TITLE
Added autologon capability to the vsts-build-agent and deployment-group artifacts

### PR DESCRIPTION
**Pull request by ShreyasRmsft**

### CODE REVIEW

File: Artifacts/windows-deploymentgroup/Artifactfile.json [hunk 1]
```diff
@@ -46,9 +46,28 @@
...
+        "RunAsAutoLogon": {
+            "type": "bool",
+            "displayName": "Run As Auto Logon",
+            "description": "Enable autologon to run the build agent in interactive mode for UI testing.",
+            "defaultValue": false
+        },
+        "windowsLogonAccount": {
+            "type": "string",
+            "displayName": "Account Name",
+            "description": "The Windows logon account name which will run the agent service or process.",
```
Comment (nigurr) [inline Artifacts/windows-deploymentgroup/Artifactfile.json]: Check with Prachi for strings

Comment (ShreyasRmsft) [reply Artifacts/windows-deploymentgroup/Artifactfile.json]: Will do

**Comment by ShreyasRmsft** [comment]
@kmkumaran @leovms Please review

File: Artifacts/windows-vsts-build-agent/Artifactfile.json [hunk 2]
```diff
@@ -39,16 +39,22 @@
...
+      "defaultValue": false
+    },
     "windowsLogonAccount": {
-      "type": "string",
-      "displayName": "Service Account Name",
-      "description": "The Windows logon account name which will run the agent service.",
-      "defaultValue": "NT AUTHORITY\\NetworkService"
+        "type": "string",
+        "displayName": "Account Name",
+        "description": "The Windows logon account which will run the agent.<br>When autologon is enabled, agent runs as a process in the context of the specified user.",
```
[the remark this answers is not in the document]
Comment (ShreyasRmsft) [reply Artifacts/windows-vsts-build-agent/Artifactfile.json]: image

It does work, it's only after testing that i left it in.

**Pull request merged**